[2024-08-27T13:04:30.810+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:04:30.811+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:04:30.843+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:04:30.844+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:04:30.870+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:04:30.882+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:04:30.895+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:04:30.896+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-27T13:04:30.912+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): calculate_exchange_rate_change> on 2024-08-27 12:00:00+00:00
[2024-08-27T13:04:30.916+0000] {standard_task_runner.py:57} INFO - Started process 216 to run task
[2024-08-27T13:04:30.919+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'currency_exchange_dag', 'calculate_exchange_rate_change', 'scheduled__2024-08-27T12:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_dag.py', '--cfg-path', '/tmp/tmp7agby0jz']
[2024-08-27T13:04:30.924+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask calculate_exchange_rate_change
[2024-08-27T13:04:30.939+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:04:30.972+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:04:30.973+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:04:30.974+0000] {task_command.py:410} INFO - Running <TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [running]> on host ff5f64dee89e
[2024-08-27T13:04:30.985+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:04:31.048+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_exchange_dag' AIRFLOW_CTX_TASK_ID='calculate_exchange_rate_change' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-27T12:00:00+00:00'
[2024-08-27T13:04:31.054+0000] {exchange_rate_dag.py:69} INFO - Exchange rate changes calculated successfully.
[2024-08-27T13:04:31.055+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-27T13:04:31.065+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=currency_exchange_dag, task_id=calculate_exchange_rate_change, execution_date=20240827T120000, start_date=20240827T130430, end_date=20240827T130431
[2024-08-27T13:04:31.091+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-27T13:04:31.109+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-27T13:18:55.670+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:18:55.671+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:18:55.703+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:18:55.704+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:18:55.728+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:18:55.740+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:18:55.749+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:18:55.749+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-27T13:18:55.765+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): calculate_exchange_rate_change> on 2024-08-27 12:00:00+00:00
[2024-08-27T13:18:55.769+0000] {standard_task_runner.py:57} INFO - Started process 200 to run task
[2024-08-27T13:18:55.772+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'currency_exchange_dag', 'calculate_exchange_rate_change', 'scheduled__2024-08-27T12:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_dag.py', '--cfg-path', '/tmp/tmpde79iv64']
[2024-08-27T13:18:55.774+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask calculate_exchange_rate_change
[2024-08-27T13:18:55.789+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:18:55.829+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:18:55.829+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:18:55.832+0000] {task_command.py:410} INFO - Running <TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [running]> on host 0b490f25fde1
[2024-08-27T13:18:55.846+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T13:18:55.917+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_exchange_dag' AIRFLOW_CTX_TASK_ID='calculate_exchange_rate_change' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-27T12:00:00+00:00'
[2024-08-27T13:18:55.923+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-27T13:18:55.933+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=currency_exchange_dag, task_id=calculate_exchange_rate_change, execution_date=20240827T120000, start_date=20240827T131855, end_date=20240827T131855
[2024-08-27T13:18:55.985+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-27T13:18:56.011+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-27T13:43:19.141+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:43:19.142+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:43:19.175+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:43:19.175+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:43:19.205+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-08-27T13:43:19.216+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:43:19.228+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [queued]>
[2024-08-27T13:43:19.229+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-27T13:43:19.244+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): calculate_exchange_rate_change> on 2024-08-27 12:00:00+00:00
[2024-08-27T13:43:19.248+0000] {standard_task_runner.py:57} INFO - Started process 65 to run task
[2024-08-27T13:43:19.251+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'currency_exchange_dag', 'calculate_exchange_rate_change', 'scheduled__2024-08-27T12:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_dag.py', '--cfg-path', '/tmp/tmp3wpp7u8k']
[2024-08-27T13:43:19.252+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask calculate_exchange_rate_change
[2024-08-27T13:43:19.269+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-27T13:43:19.312+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-27T13:43:19.312+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-27T12:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-27T13:43:19.314+0000] {task_command.py:410} INFO - Running <TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-27T12:00:00+00:00 [running]> on host d374469d5083
[2024-08-27T13:43:19.326+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-08-27T13:43:19.404+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_exchange_dag' AIRFLOW_CTX_TASK_ID='calculate_exchange_rate_change' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-27T12:00:00+00:00'
[2024-08-27T13:43:19.409+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-27T13:43:19.421+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=currency_exchange_dag, task_id=calculate_exchange_rate_change, execution_date=20240827T120000, start_date=20240827T134319, end_date=20240827T134319
[2024-08-27T13:43:19.465+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-27T13:43:19.500+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
